File: app/src/main/java/com/alvinhkh/buseta/kmb/model/network/KmbEtaRes.java
package com.alvinhkh.buseta.kmb.model.network;


import com.alvinhkh.buseta.kmb.model.KmbEta;
import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;

public class KmbEtaRes {

  @SerializedName("responsecode")
  public Integer responsecode;

  @SerializedName("response")
  public ArrayList<KmbEta> etas;

  @SerializedName("generated")
  public Long generated;

  @SerializedName("updated")
  public Long updated;

  public String toString() {
    return "KmbEtaRes{responsecode=" + this.responsecode + ", response=" + this.etas
        + ", generated=" + this.generated + ", updated=" + this.updated + "}";
  }

}

File: app/src/main/java/com/alvinhkh/buseta/kmb/model/network/KmbEtaRes.kt
package com.alvinhkh.buseta.kmb.model.network


import com.alvinhkh.buseta.kmb.model.KmbEta
import com.google.gson.annotations.SerializedName

data class KmbEtaRes(
        @SerializedName("routeNo")
        var routeNo: String? = null,

        @SerializedName("bound")
        var bound: Int? = null,

        @SerializedName("service_type")
        var serviceType: Int? = null,

        @SerializedName("seq")
        var seq: Int? = null,

        @SerializedName("responsecode")
        var responseCode: Int? = null,

        @SerializedName("generated")
        var generated: Long? = null,

        @SerializedName("updated")
        var updated: Long? = null,

        @SerializedName("eta")
        var etas: ArrayList<KmbEta>? = null
)